cdap-unit-test/src/main/java/io/cdap/cdap/test/TestBase.java

```diff
@@ -369,6 +371,8 @@ protected void configure() {
     provisioningService = injector.getInstance(ProvisioningService.class);
     provisioningService.startAndWait();
     metadataSubscriberService.startAndWait();
+    appFabricServer = injector.getInstance(AppFabricServer.class);
+    appFabricServer.startAndWait();
```

Comment:
not really sure how things were working before if app fabric was never getting started before. Unless I'm missing something, it doesn't seem like the changes in this PR would affect TestBase?

Reply:
In unit tests, we always use NOSQL thus will always exercise the remote call code path. 

Removed for now.

Thought some other preview tests are failing due to AppFabric being not up. Let me run all tests again and see if anything is broken without this.

Reply:
@albertshau Without starting AppFabric, PreviewDataPipelineTest fails as we use NOSQL in test and preview in this case fetch artifact, preferences from appfabric. Previously preview just access local levelDB directly. Need to bring back this starting up appfabric.